www/json/trip_confirm_options.json.sample

```diff
@@ -6,8 +6,10 @@
             "ALL": {"range": [0, -1], "mets": 4.9}
        }, "co2PerMeter": 0.00728},
        {"text":"Scooter share","value":"scootershare", "met_equivalent": "IN_VEHICLE", "co2PerMeter": 0.00894},
-       {"text":"Drove Alone","value":"drove_alone", "met_equivalent": "IN_VEHICLE", "co2PerMeter": 0.22031},
-       {"text":"Shared Ride","value":"shared_ride", "met_equivalent": "IN_VEHICLE", "co2PerMeter": 0.11015},
+       {"text":"Gas Car Drove Alone","value":"drove_alone", "met_equivalent": "IN_VEHICLE", "co2PerMeter": 0.22031},
+       {"text":"Gas Car Shared Ride","value":"shared_ride", "met_equivalent": "IN_VEHICLE", "co2PerMeter": 0.11015},
```

Comment:
```suggestion
       {"text":"Gas Car Drove Alone","value":"gas_car_drove_alone", "met_equivalent": "IN_VEHICLE", "co2PerMeter": 0.22031},
       {"text":"Gas Car Shared Ride","value":"gas_car_shared_ride", "met_equivalent": "IN_VEHICLE", "co2PerMeter": 0.11015},
```

Reply:
If this were a brand-new system, this would make sense. However, we are updating an existing system.
If we change this to `gas_car_drove_alone` and the user tried to display a trip from last week which was labeled as `drove_alone`, then it would not find any match.

Keeping it as `drove_alone` gives us backwards compatibility.

We can think about how to deal with backwards-compat programmatically. For example, we can write a script to change all `drove_alone` to `gas_car_drove_alone` on the server. It's a little bit tricky because it is tricky to guarantee that the server and the phone are in sync since phone apps have to go through the app store review process.

There are ways around all of this (hardcoded, temporary hack that handles both modes on the phone, scheduling release at a particular date/time), but not sure I want to tackle that right now.

Reply:
This update will not make things worse, since we always had `drove_alone` strings. We can make the change later as part of a separate release. File an issue, and maybe you can tackle it as a "small" change.